### ISSUE TITLE
Built-in GTK+ clipboard causes NeovimGtk to stop working.

### ISSUE THREAD

**Issue by MeanEYE** [daa84/neovim-gtk-flatpak#6]
After enabling `g:GuiInternalClipboard` application just hangs on startup. Sorry for the lack of debug information, but I am unable to find any sort of logging for Flatpak.

**Comment by daa84** [comment]
looks like flatpak specific problem.

**Comment by MeanEYE** [comment]
How so? I have other applications which are installed through Flatpak which have no issues with clipboard.

**Comment by daa84** [comment]
Yes, this is issue of neovim-gtk but reproduced only in flatpak, i try to fix it :smiley:

**Comment by MeanEYE** [comment]
Thanks for looking into it. And I have to add this is the best Neovim client yet. You did a very good job!
